Formula/deark.rb

```diff
@@ -0,0 +1,23 @@
+require "base64"
+
+class Deark < Formula
+  desc "File conversion utility for older formats"
+  homepage "http://entropymine.com/deark/"
+  url "http://entropymine.com/deark/releases/deark-1.4.5.tar.gz"
+  sha256 "2dfe61cc7bfa927e1702b2312b3edb9e5f9e67bfe460f5da3f82652f163e31e0"
+
+  def install
+    system "make"
+    bin.install "deark"
+  end
+
+  test do
+    (testpath/"test.gz").write(Base64.decode64(<<~EOS
```

Comment:
The syntax in https://github.com/Homebrew/homebrew-core/pull/26698#issuecomment-381785290 might be preferable, to avoid those dangling parens.